config/locales/defaults/en.yml

```diff
@@ -4,6 +4,10 @@ en:
     calculated_google_analytics: 'calculated from Google Analytics'
     feedback_explorer: 'Feedback Explorer'
     none: false
+  edit_content:
+    edit_page: 'Make changes to the page'
+    edit_link: 'Edit in %{publishing_app}'
+    publisher_edit_link: 'Request a change to mainstream content'
```

Comment:
I feel content designers will expect these to be in `locales/views/metrics/en.yaml`

Reply:
I moved them out of the metrics specific file when I moved the code to the helper, but I'm fine with keeping them in the metrics related file as well. I've moved them back now.